tests/phpunit/elementor/modules/atomic-widgets/test-atomic-widget-base.php

```diff
@@ -316,8 +226,23 @@ public function test_get_props_schema__is_serializable() {
 		}', $serialized );
 	}
 
-	public function test_get_props_schema() {
+	public function test_get_props_schema__not_throwing_when_the_schema_is_invalid() {
```

Comment:
I don't think this test is necessary

Reply:
it's to make sure that the `get_props_schema` doesn't validate the schema each time you call it, to ensure it's not running in the frontend

Reply:
There is no situation in which the method will throw an exception, so I think you should delete it.